src/orga/governance.rst

```diff
@@ -189,6 +212,7 @@ To call for a standard vote, here is a template PR comment:
 
     * Sensitive
     * 50% Majority to pass
+    * Quorum rules: standard, accelerated, or time-out
```

Comment:
Should we be accelerating sensitive votes?

Reply:
For staged recipes, I figured this would be fine. I did not apply this to other sensitive votes.